lib/messager.rb

```diff
@@ -0,0 +1,23 @@
+require 'twilio-ruby' 
+
+class Messager
+
+  def send_message(txt_message) 
+    config
+    @client.account.messages.create({
+      :from => '+441143032569', 
+      :to => '+447832366992', 
+      :body => txt_message,
+    })
+  end  
+  
+  private
+  def config
+    account_sid = 'ACa742df3682aa80d6d524a052cf5b0013'            # My credentials are saved as environment variables.    
```

Comment:
You should change your API keys as these ones are now publicly accessible on GitHub